### DISCUSSION TITLE
Kudos!

### DISCUSSION THREAD

**Discussion by foxxcomm** [zwave-js/zwave-js-ui#3675] [Show and tell]
Just wanted to send huge Kudo's to the Z-Wave JS and Z-Wave JS UI Developers on the addition of Z-Wave LR support and NVM backup improvements.

We just tested upgrades of our previous Z-Wave network controllers (Homeseer Z-Net 500-Chipset and Homeseer Z-Net 700-Chipset) to the 800 Chipset GPIO Module (ZAC93). NVM backup from our 700 Chipset controller (Homeseer Z-Net) with restore to our 800 Chipset controller worked flawlessly. NVM Backup from the 500 controller with restore directly to the 800 controller would not work directly, however the solution was simply to restore the 500 NVM backup first to our 700 controller, then NVM backup our 700 controller and restore NVM to the 800 controller.

We included several new Z-Wave 800 LR devices in our network in areas that were previously complete RF dead spots (no close nodes to use as repeaters). These areas are now also working flawlessly with direct Z-Wave LR connections to the controller.

Migrated Z-Wave JS UI from Home Assistant to a service running under Windows so we could have several instances of Z-Wave JS UI running at once (on different Z-Wave networks). Z-Wave JS UI's built-in Ser2Net functionality working flawlessly connecting to the Z-Net controllers via Ethernet and feeding our multiple Z-Wave production, debug and test networks into the Home Assistant Z-Wave Integration via Websocket. Also working flawlessly!

Bravo!

Thank You!

**Comment by robertsLando** [comment]
Thanks @foxxcomm 🙏🏻❤️

**Comment by AlCalzone** [comment]
Thanks @foxxcomm!

About the 500->700->800 migration, I'm happy to take a look why the additional step was necessary, since it shouldn't.
Can you send me the 500 series backup and a backup of the 800 series before restoring the other backup via email?

**Comment by foxxcomm** [comment]
AlCalzone –

What email address to do want them zipped and sent to?

Be happy to collect these for you.

Thanks

**Reply by AlCalzone** [reply]
Just use the one from my [GitHub profile](https://github.com/AlCalzone)
